### ISSUE TITLE
Some values are exported incorrectly when saving XML using Eclipse Plugin

### ISSUE THREAD

**Issue by Fanfouna** [spotbugs/spotbugs#2031]
Some values (like _cpu_seconds_ and _clock_seconds_) are not exported correctly when manually saving the Bug Collection as an XML file using the Eclipse Plugin. Afaik, these two values are supposed to show how long the analysis of the project took, but the time gets counted until you press the _"Save XML"_ button. So if I export the same Bug Collection twice, the second time some values are higher, as shown in the below example. And of course this also means that the exported values are different from those displayed when the _"Open Analysis Results in Editor"_ option is picked (which reads from the automatically generated project-name.fbwarnings.xml file) 
Tldr: When manually exporting the xml results, the values in the exported xml should be identical to the ones in the project-name.fbwarnings.xml file. 

**Example from manually exported XML:**
`<FindBugsSummary timestamp="Mon, 25 Apr 2022 12:04:47 +0200" total_classes="608" referenced_classes="1562" total_bugs="573" total_size="31121" num_packages="67" java_version="11.0.14.1" vm_version="11.0.14.1+1-b2043.25" cpu_seconds="105.67" clock_seconds="54.55" peak_mbytes="1286.93" alloc_mbytes="1018.00" gc_seconds="1.17" priority_3="266" priority_2="246" priority_1="61"></FindBugsSummary>`
**Same Bug Collection, exported ~3 minutes later**
`<FindBugsSummary timestamp="Mon, 25 Apr 2022 12:04:47 +0200" total_classes="608" referenced_classes="1562" total_bugs="573" total_size="31121" num_packages="67" java_version="11.0.14.1" vm_version="11.0.14.1+1-b2043.25" cpu_seconds="123.22" clock_seconds="240.66" peak_mbytes="1422.30" alloc_mbytes="1018.00" gc_seconds="1.27" priority_3="266" priority_2="246" priority_1="61"> </FindBugsSummary>`
